include/boost/multiprecision/cpp_dec_float.hpp

```diff
@@ -382,11 +382,18 @@ class cpp_dec_float
    {
       using default_ops::eval_add;
       using default_ops::eval_multiply;
-      static constexpr unsigned          bit_shift = sizeof(unsigned long long) * CHAR_BIT;
+
+      static constexpr unsigned     bit_shift = sizeof(unsigned long long) * CHAR_BIT;
       static constexpr uint128_type mask      = (static_cast<uint128_type>(1u) << bit_shift) - 1;
-      unsigned                           shift     = bit_shift;
```

Comment:
That must have been a cut-and-paste error, variable shift is unused and can be deleted rather than cast to void.

Other than that, this all looks great to me!

Reply:
I forgot this in my recent push to green. I am going to commit and merge to develop anyway and cycle back to this trivial removal later.